torch/_torch_docs.py

```diff
@@ -4365,6 +4365,48 @@ def merge_dicts(*dicts):
             [1.0311, 0.3901, 0.5049]])
 """)
 
+add_docstr(torch.movedim,
+           r"""
+movedim(input, src, dst) -> Tensor
```

Comment:
In numpy, their parameter names are `numpy.moveaxis(a, source, destination)`. Should we try to match their convention? If so, these arguments should be called `movedim(tensor, source, destination)`. @mruberry what do you think?

Reply:
Naming the last two arguments "source" and "destination" sounds good, although NumPy's naming for positional arguments is very inconsistent and we generally consider changing positional names to be "safe."